EosPayload/drivers/position_aware_logging_driver.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+from EosPayload.lib.base_drivers.position_aware_driver_base import PositionAwareDriverBase
+
+
+class PositionAwareLoggingDriver(PositionAwareDriverBase):
+
+    def setup(self) -> None:
+        super().setup()
+        self.register_thread('position-logging', self.device_command)
+
+    def device_command(self, logger: logging.Logger) -> None:
+        while True:
+            self.thread_sleep(logger, 10)
+            try:
+                if self.latest_position is not None:
+                    self.data_log([str(self.latest_position.timestamp),
+                                   str(self.latest_position.latitude),
+                                   str(self.latest_position.longitude),
+                                   str(self.latest_position.altitude),
+                                   str(self.latest_position.speed),
+                                   str(self.latest_position.number_of_satellites),
+                                   str(self.latest_position.valid),
+                                   str(self.latest_position.flight_state.name)])
+                else:
+                    self.data_log(["no position"])
```

Comment:
data_log uses csv, are you sure you don't need to fill the missing cols?

Reply:
That would arguably be better, but the default behavior for PositionAwareDriverBase is a Position with all none values, so the else case should probably never happen and I want to be able to distinguish if it does and can just parse it out.